docs/SlavaUkraini/dialog.js

```diff
@@ -0,0 +1,34 @@
+var SESSION_KEY = 'ukraini-dialog-session';
+var ONE_DAY_MILLI_SEC = 24 * 60 * 60 * 1000;
+
+function openSlavaUkrainiDialog() {
+
+	if (sessionStorage) {
+		var session = sessionStorage.getItem(SESSION_KEY);
+		// open the dialog only every 24 hours
+		if (session && Date.now() - session < ONE_DAY_MILLI_SEC) {
+			return;
+		}
+	}
+
+	var html = '<div class=\'container iframe-container\'><span class=\'close-dialog\' aria-label=\'close\'>&times;</span><iframe src=\'/SlavaUkraini/\' /></div>';

```

Comment:
For keyboard-friendly dialog:

1. [`autofocus`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus) the dialog
2. Change the DOM order of iframe and the close button

Reply:
`autofocus` does only work if the element is already existing while page loading. I used now a JS solution `focus()`.

Is there a way to make a `button` / `a` element clickable over enter / space without adding keypress listeners?

Reply:
> autofocus does only work if the element is already existing while page loading

Ok, otherwise could not be added/removed from DOM but instead have its visibility toggled?

> Is there a way to make a `button` / `a` element clickable over enter / space without adding keypress listeners?

`<button type="submit">` inside a `<form method="dialog">` is one way. But [`dialog`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) requires a [polyfill](https://github.com/GoogleChrome/dialog-polyfill) due to current [support](https://caniuse.com/dialog).